service/instance.py

```diff
@@ -1006,7 +1000,7 @@ def _boot_volume(driver, identity, copy_source, size, name, userdata, network,
     return (new_instance, token, password)
 
 
-def _launch_volume(driver, identity, volume, size, userdata_content, network,
+def _launch_volume(driver, identity, volume, size, name, userdata_content, network,
```

Comment:
This is called from:
/server/instance.py 967
967       instance, token, password = _launch_volume(
968         driver, identity, volume, size,
969         name, userdata, network, **kwargs)

Added name to the call signature here as it is referenced in the function.